deploy/manifests/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cert-manager
```

Comment:
note: the other similar file has `---` in it; a new file is the easiest backwards compatible way I could think of here. Eventually, the other file will go away with bazel.